src/main/java/com/saucelabs/ci/sauceconnect/SauceConnectFourManager.java

```diff
@@ -255,7 +255,13 @@ protected String[] generateSauceConnectArgs(String[] args, String username, Stri
     }
 
     protected String[] addExtraInfo(String[] args) {
-        String[] result = joinArgs(args, "--extra-info", "{\"runner\": \"jenkins\"}");
+        String[] result;
+        OperatingSystem operatingSystem = OperatingSystem.getOperatingSystem();
+        if (operatingSystem == OperatingSystem.WINDOWS) {
+            result = joinArgs(args, "--extra-info", "{\\\"runner\\\": \\\"jenkins\\\"}");
+        } else {
+            result = joinArgs(args, "--extra-info", "{\"runner\": \"jenkins\"}");
```

Comment:
Although I'm interested in this fix merged ASAP I'd recommend adding a new test mocking the `getOperatingSystem` response with `Windows` and others. This way you can validate the results and avoid any regression issue. Thoughts @amckenzie132 ?!

Reply:
actually CI builds are run on 3 platforms

Reply:
But this problem happens in Windows environment only. Unix environment (which includes Linux and MacOS) seems to be ok.

Reply:
Hmmm... those unit tests that were updated are executed in different environment.
In this case, we would have some coverage. Sounds good.